ERCS/erc-7730.md

```diff
@@ -1396,6 +1396,42 @@ The binding `context` mitigates misuse by specifying exactly which structured da
 
 Curation and registry of ERC-7730 files are out of scope for this specification, but any external registry or wallet ecosystem MUST assume it will be targeted. A secure registry should (a) require cryptographically verifiable provenance and attestations for each ERC-7730 file and its maintainer, (b) keep a public, tamper-evident history of submissions, approvals, and revocations, (c) implement a mechanism to credibly link contract ownership or authority to the submitted file, and (d) adopt a clear governance model with multi-party sign-off and automated monitoring to detect anomalies or mass poisoning attempts. These measures mitigate attacks on ERC-7730 files in transit and make registry compromise significantly harder without constraining this ERC itself.
 
+### External Data Resolution
+
+ERC-7730 formatting relies on resolving various external data sources to provide human-readable information. This resolution introduces trust assumptions that wallets must understand when showing ERC-7730 information:
+
+**Trust Assumptions:**
+
+The transition from transaction data to human readable data requires trust in external data sources to resolve some parts of the presentation layer. Without the resolution, it's not readable and missed the whole point of the ERC, with the resolution it adds a risk factor:
+- **Token metadata** (tickers, decimals) A token address translated to a token ticker and proper decimals applied to the `tokenAmount`.
+- **NFT collection names** and token metadata for the `nftName` format
```

Comment:
In addition, all name sources for addresses - typically used by `addressName` formatter  (ENS, wallet specific names, ...) must be trusted and users must understand they trust their wallet to retrieve those safely